### PR TITLE
Fix issues found by the PhpStorm code inspector

### PR DESCRIPTION
**Pull request by leofeyer**

### CODE REVIEW

File: core-bundle/src/Resources/contao/elements/ContentDownloads.php [hunk 1]
```diff
@@ -266,14 +265,14 @@ protected function compile()
 		{
 			default:
 			case 'name_asc':
-				uksort($files, static function (): int
+				uksort($files, static function ($a, $b): int
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/elements/ContentDownloads.php:269]: @fritzmg How can this ever have worked in Contao 4.9?

File: calendar-bundle/src/Resources/contao/classes/Events.php [hunk 2]
```diff
@@ -202,7 +202,6 @@ protected function getAllEvents($arrCalendars, $intStart, $intEnd, $blnFeatured
 	 */
 	protected function addEvent($objEvents, $intStart, $intEnd, $intBegin, $intLimit, $intCalendar)
 	{
-		/** @var PageModel $objPage */
 		global $objPage;
```
Comment (ausi) [inline calendar-bundle/src/Resources/contao/classes/Events.php]: How does the IDE now know about the type of `global $objPage`?

File: core-bundle/src/Resources/contao/elements/ContentDownloads.php (hunk 1 above)
Comment (ausi) [reply core-bundle/src/Resources/contao/elements/ContentDownloads.php:269]: The sort function would have always returned `0` so I think the array would not have been sorted at all.

Comment (leofeyer) [reply core-bundle/src/Resources/contao/elements/ContentDownloads.php:269]: So we should definitely backport this. 😄

File: calendar-bundle/src/Resources/contao/classes/Events.php (hunk 2 above)
Comment (leofeyer) [reply calendar-bundle/src/Resources/contao/classes/Events.php]: I don't really know how it knows, I just know that is knows. 😄

Comment (m-vo) [reply calendar-bundle/src/Resources/contao/classes/Events.php]: I think this is a false positive. If I remove the annotation, PHPStorm won't complain, but it also won't give me any autocompletion or type info anymore.

Comment (leofeyer) [reply calendar-bundle/src/Resources/contao/classes/Events.php]: That's really strange. Seems I have a plugin that can do it:

Comment (leofeyer) [reply calendar-bundle/src/Resources/contao/classes/Events.php]: I have reverted the `global $objPage` changes in 12840066dfbaaede838eab0bd484133249a3db54, although I still wonder why my PhpStorm was able to infer the type and yours was not. 🤷🏻‍♂️

Comment (m-vo) [reply calendar-bundle/src/Resources/contao/classes/Events.php]: In your last image `$objUploader` can be inferred from `$objUploader = new FileUpload()`.

I was referring to the `global $objPage` annotations before.

image

vs

image

**Review by ausi — APPROVED** [review]

**Review by m-vo — APPROVED** [review]

**Pull request merged**